### PR TITLE
Added robots.txt and sitemap.xml

### PR DESCRIPTION
**Pull request by dchen97**
Added `robots.txt` and `sitemap.xml` to `static/assets` folder in order to improve SEO. Added appropriate routes as well to `app.yaml`.

### CODE REVIEW

File: app/static/assets/robots.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# All robots are allowed
+User-agent: *
+Disallow:
+
+# Sitemap files
+#Sitemap: ./sitemap.xml
```
Comment (seanlip) [inline app/static/assets/robots.txt]: Just checking, did you mean to comment this line out? I looked at some sample robots.txt files and they don't do that...

Also, do we need to use the absolute URL, or is the relative URL also OK?

File: app/static/assets/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <url>
+        <loc>https://www.oppiafoundation.org/volunteer</loc>
+        <priority>0.80</priority>
+    </url>
+    <url>
+        <loc>https://www.oppiafoundation.org/donate</loc>
+        <priority>0.80</priority>
+    </url>
+
+</urlset>
```
Comment (seanlip) [inline app/static/assets/sitemap.xml]: Missing newline at EOF.

File: app/static/assets/robots.txt (hunk 1 above)
Comment (dchen97) [reply app/static/assets/robots.txt]: I just checked and adjusted it. It was commented out when I got the file from Asma, but I just uncommented it.

File: app/static/assets/sitemap.xml (hunk 2 above)
Comment (dchen97) [reply app/static/assets/sitemap.xml]: Fixed

**Pull request merged**